src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -240,23 +251,25 @@ const ExtraSmallAuctionItem: SFC<Props> = props => {
         )}
       </Flex>
       <Flex ml={2} flexDirection="column" justifyContent="center" width="100%">
+        <Text variant="xs" color="black100">
+          {title}
+          {title && date_text && ", "}
+          {date_text}
+        </Text>
+        <Text variant="xs" color="black60">
+          Sold on {dateOfSale}
+        </Text>
+        <Spacer size="10px" />
```

Comment:
I'd do `mt={1}` though we should probably make this accept space props. (`size` is a shortcut for width & height)

Reply:
oh snap. ok I can fix that. I do find it weird to give mt or ml or mr or mb when the thing is 0 size and a margin. in my mind a spacer has a size or width/height. in that way, size made sense.

Reply:
Yeah I agree; though I think we should give it a more thoughtful API. The thing is that size/width/height doesn't use the spacing scale so it's easy to create spacers that don't conform.